person.rb

```diff
@@ -19,6 +20,10 @@ def validate_name
     @name = @corrector.correct_name(name)
   end
 
+  def add_rental(rental)
+    rentals.push(rental)
+  end
```

Comment:
- [ ] Kindly modify your `add_rental` method to create a Rental object instead, as described above. Take note of the arrangement of parameters too. You would need to import the Rental class here for this to work.